subsys/emds/Kconfig

```diff
@@ -9,6 +9,7 @@ menuconfig EMDS
 	select EXPERIMENTAL
 	depends on PARTITION_MANAGER_ENABLED
 	depends on FLASH_MAP
+	depends on CRC
```

Comment:
I don't understand how this is related to the rest of this PR? What does this do?

Reply:
This is a dependency that has been forgotten. The emds_flash file is dependent on CRC, but it has always been added because we most of the time just have added it or it has been added through NVS. But the dependency has always been there, but it has not been showed explicit. I discovered it now when running test without settings, and then I got compilations errors. So I thought it was better to added so people get dependency error instead of compilation error.